modules/torch_compile.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>I would instead show the mean of the times.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Hi Eric, since for the compile mode the first iteration may take much more time than eager mode, so we may need more iteration to see the speedup from the mean value, so here I show the median value follow the behavior from the torch repo. The end-to-end result show the comparison from the total time.</p>

Reply:
The issue is that median in this case just happens to be showing what you want, but because the values aren't a statistical distribution it doesn't really mean much mathematically. The average of the compiled times should be much lower than uncompiled, that's what would actually be meaningful for the actual overall runtime.

Reply:
Ok, will update this part, thanks.

Reply:
Using mean instead.